docs/scenarios/parametric.md

```diff
@@ -112,7 +112,19 @@ TEST_LIBRARY=java ./run.sh test_span_sampling.py::test_single_rule_match_span_sa
 
 #### PHP
 
-If you are seeing DNS resolution issues when running the tests locally, add the following config to the Docker daemon:
+##### To run with a custom build
+
+- Download `datadog-setup.php` in `build packages/package extension` job on CI
```

Comment:
To avoid ambiguity, I would use 'build_packages' instead of 'build packages' to clarify that it is not under the 'build' workflow. I obviously know this, but it may not be for someone who isn't familiar with our CI 😃